student-test/assignment1/Test_validate_username_Runner.c

```diff
@@ -0,0 +1,85 @@
+/* AUTOGENERATED FILE. DO NOT EDIT. */
```

Comment:
@SundarKrishnakumar 
again these files should be ignored, see https://github.com/cu-ecen-aeld/aesd-assignments/blob/master/.gitignore#L3
You typically shouldn't check in autogenerated files into your repo.